examples/neff_shuffle_test.go

```diff
@@ -0,0 +1,106 @@
+package examples
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/kyber/v3"
+	kproof "go.dedis.ch/kyber/v3/proof"
+	"go.dedis.ch/kyber/v3/shuffle"
+)
+
+/*
```

Comment:
We have all comments as `//` and don't use the `/*` syntax. So for uniformity I propose to change the comment format.

Reply:
That's not the case for the majority of files in this package, which is why I used this comment format.

Reply:
True - but I'm really not used to see this in go. So keep it like this.